geoportailv3/static/js/draw/featurepopupservice.js

```diff
@@ -159,6 +163,8 @@ app.FeaturePopup.prototype.show = function(feature, opt_anchor) {
   goog.array.forEach(headers, function(element) {
     this.setDraggable(angular.element(element));
   }, this);
+  // Enable the dropdown menu
+  $('.dropdown-toggle')['dropdown']();
```

Comment:
It probably depends on if bootsrap is already initialized when the element is created or not.